internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr.go

```diff
@@ -112,7 +113,42 @@ func (k *KafkaManager) Reconcile() []error {
 		}
 	}
 
-	// cleaning up expired qkafkas
+	validSuspensionStatuses := []constants.KafkaStatus{
```

Comment:
Could we refactor the logic in its own method?

Reply:
Done! Added a `CanBeSuspended` to the `KafkaRequest` object